### PR TITLE
Update info about new command generating self signed ssl certificate

### LINKED ISSUE #138: Our SSL certificates do not have subjectAltName
@jiivan Reports that the certificates we have generated for the testnet cluster (and likely for the other clusters too) produce the following warning:

```
SubjectAltNameWarning: Certificate for test.concent.golem.network has no `subjectAltName`, falling back to check for a `commonName` for now. This feature is being removed by major browsers and deprecated by RFC 2818. (See https://github.com/shazow/urllib3/issues/497 for details.)
  SubjectAltNameWarning
```

- Update the [command in our README](https://github.com/golemfactory/concent-deployment#generating-self-signed-certificates)
- Generate new certificates
- Post updated certificates on the wiki (https://github.com/golemfactory/concent-deployment-values/wiki/concent-status)
- Find out if there are any other issues we should know about. Post anything interesting you find in the comments below.

### PR DESCRIPTION
**Pull request by bartoszbetka**
Resolves #138

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,16 +70,44 @@ It's best if your certificates are signed by a Certificate Authority (CA) becaus
...
-    -keyout nginx-proxy-ssl.key \
-    -out    nginx-proxy-ssl.crt
+openssl req                      \
+    -x509                        \
+    -nodes                       \
+    -days 365                    \
+    -sha256                      \
+    -newkey rsa:2048             \
+    -out nginx-proxy-ssl.crt     \
+    -keyout nginx-proxy-ssl.key  \
```
Comment (cameel) [inline README.md]: Please restore the previous alignment. It was more readable that way.

File: README.md [hunk 2]
```diff
@@ -70,16 +70,44 @@ It's best if your certificates are signed by a Certificate Authority (CA) becaus
...
+OU           = $organization_unit
+CN           = $domain_name
+emailAddress = $email_address
+
+[ x509_ext ]
+basicConstraints     = CA:FALSE
+subjectAltName       = @alt_names
+subjectKeyIdentifier = hash
+
+[ alt_names ]
```
Comment (cameel) [inline README.md]: Please remove the unnecessary whitespace here and in other headers.

File: README.md [hunk 3]
```diff
@@ -70,16 +70,44 @@ It's best if your certificates are signed by a Certificate Authority (CA) becaus
...
+openssl req                      \
+    -x509                        \
+    -nodes                       \
+    -days 365                    \
+    -sha256                      \
+    -newkey rsa:2048             \
+    -out nginx-proxy-ssl.crt     \
+    -keyout nginx-proxy-ssl.key  \
+    -config <(                   \
+cat <<-EOF
```
Comment (cameel) [inline README.md]: Instead of creating the file inline, please just include it separately, as a template and tell the user to fill in specific fields. The way it is now it looks more complex than it really has to.

File: README.md [hunk 4]
```diff
@@ -70,16 +70,44 @@ It's best if your certificates are signed by a Certificate Authority (CA) becaus
 This is not required though.
 You can generate and use a self-signed certificate.
 
-Here's an example command that generates a 2048-bit RSA certificate valid for a year:
+Here's an example command that generates a 2048-bit RSA certificate valid for a year.
+Before execute command, set shell variables: `$country`, `$state`, `$organization`, `$organization_unit`, `$domain_name`, `$email_address`
```
Comment (cameel) [inline README.md]: Please reword:

```
Replace `$country`, `$state`, `$organization`, `$organization_unit`, `$domain_name`, `$email_address` with actual values.
```

Also, put it below the config file template, not here.

Comment (bartoszbetka) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (bartoszbetka) [reply README.md]: Done.
